lib/promoted/ruby/client/options.rb

```diff
@@ -161,17 +150,15 @@ def request_id
 
         def log_request_params
           {
-            platform_id: platform_id,
             user_info: user_info,
             timing: timing,
-            request: request,
+            request: [request],
```

Comment:
why [ ] ?

Reply:
`LogRequest.request` is a list field (annoying plural proto naming issue).